wave_matching/tests/icp_tests.cpp

```diff
@@ -81,9 +67,9 @@ TEST_F(ICPTest, nullDisplacement) {
     this->setParams(0.05f, perturb);
     // test and assert
     match_success = matcher->match();
-    result = matcher->getResult();
+    double diff = (matcher->getResult().matrix() - perturb.matrix()).norm();
     EXPECT_TRUE(match_success);
-    EXPECT_TRUE(isApprox(result, perturb, 0.1));
+    EXPECT_LT(diff, 0.1);
```

Comment:
same comment as above, change 0.1 to a threshold defined in your test fixture.  Change throughout.